### ISSUE TITLE
Saved PDF can shift text positions

### ISSUE THREAD

**Issue by genevb** [root-project/root#12749]
Not a major issue, but saving to a PDF can result in shifted text. Here is an example where the generated PDF has two printed texts offset from each other a little, while the in-ROOT graphics show the two texts well-aligned with each other. This is in ROOT 6.25/01 and appears to be independent of whether TTF are used or not 

`
void test() {
  gStyle->SetGridColor(1);
  TH2I* h1 = new TH2I("hh","test",1,-1,1,1,-1,1);
  TText* t1 = new TText();

  TCanvas* c1 = new TCanvas("c1","c1",1000,1000);
  h1->Draw();
  gPad->SetGrid();
  t1->SetTextColor(1);
  t1->SetTextAlign(32);
  t1->SetTextAngle(0);
  t1->DrawText(-0.6,-0.6,"100");
  t1->SetTextColor(2);
  t1->SetTextAlign(12);
  t1->SetTextAngle(180);
  t1->DrawText(-0.6,-0.6,"100");
  c1->SaveAs("test1.pdf");
}
`

-Gene

**Comment by couet** [comment]
Yes, there is some little difference when you try to align text between the different graphics backends ROOt has. PDF, PS PNG, and screen have their own ways to position text. The kind of small misalignment you see is due to. I can have a look but I am not sure it will be possible to perform an exact alignment (PDF has its own fonts).

**Comment by genevb** [comment]
Thanks, Olivier. I'll accept that it's inherent to PDF creation and we can close this issue. I brought it up in case it was a minor +/-1 somewhere that it shouldn't be, but I'm not going to make you chase inherent PDF issues.

-Gene

**Comment by couet** [comment]
It is something we see under some other conditions. Not exactly these though. I will close but before I'll have a look before anyway. Thanks!
